### ISSUE TITLE
When selecting interface on Mac OS Monterey; devices are not discovered

### ISSUE THREAD

**Issue by trajche** [rristov60/XBacnet#6]
I ran into another issue when using the app on Mac (Intel).

When using 0.0.0.0 as an interface (all), I can see all the BACnet devices. 

However, when using the specific interface for the LAN port (in my case 10.100.10.x) it does not discover them.  It seems to work in a Windows environment.

**Comment by rristov60** [comment]
Thank you for opening the issue. 

This is a subject of investigation, I'll share an answer for this as soon as possible.

**Comment by rristov60** [comment]
@trajche I finally managed to find the core of the issue. The issue was with the selected interface not being properly handled by the client. The issue is being resolved and I am working on new release where this issue will be gone. I will ping you once the new release is finished and published :))

**Comment by rristov60** [comment]
Dear Trajche, this has been patched with the latest patch. I have in mind to do some security patches and some optimizations for the newest release, so I will notify you once available!
